app/script/view_model/content/TitleBarViewModel.js

```diff
@@ -80,7 +80,9 @@ z.viewModel.content.TitleBarViewModel = class TitleBarViewModel {
     });
 
     this.supportsVideoCall = ko.pureComputed(() => {
-      return this.showCallControls() ? this.conversationEntity().supportsVideoCall(true) : false;
+      return this.showCallControls()
+        ? this.userRepository.isTeam() && this.conversationEntity().supportsVideoCall(true)
+        : false;
```

Comment:
Seems like this causes a regression. Probably it is best to move the `isTeam` check to `supportsVideoCall()` to check only in case of groups and base it on the self user there.